src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
```

Comment:
The Rectangle class should not extend Figure directly if Figure is intended to be an abstract class to set behavior for classes. Consider using an interface to define common behavior instead, if applicable.